movie.py

```diff
@@ -108,7 +115,64 @@ def create_table():
             break
         time.sleep(3) 
     print('Table created!\n')
-    return response
+    return table_name
+
+def update_table(content,table_name):
+    for item in content:
+        response = g_ddb_client.update_item(
+        TableName='{}'.format(table_name),
+        Key={
+            'imdbID': {'S': '{}'.format(item['imdbID'])}
+        },
+        UpdateExpression='SET Title = :title, \
+            MovieYear = :year, \
+            Rated = :rated, \
+            Released = :released, \
+            Runtime = :runtime, \
+            Genre = :genre, \
+            Director = :director, \
+            Writer = :writer, \
+            Actors = :actors, \
+            Plot = :plot, \
+            Lang = :lang, \
+            Country = :country, \
+            Awards = :awards, \
+            Poster = :poster, \
+            Metascore = :metascore, \
+            imdbRating = :imdbrating, \
+            imdbVotes = :imdbvotes, \
+            Content_Type = :content_type, \
+            DVD = :dvd, \
+            BoxOffice = :boxoffice, \
+            Production = :production, \
+            Website = :website',
+        ExpressionAttributeValues={
+            ':title': {'S': '{}'.format(item['Title'])},
+            ':year': {'S': '{}'.format(item['Year'])},
+            ':rated': {'S': '{}'.format(item['Rated'])},
+            ':released': {'S': '{}'.format(item['Released'])},
+            ':runtime': {'S': '{}'.format(item['Runtime'])},
+            ':genre': {'S': '{}'.format(item['Genre'])},
+            ':director': {'S': '{}'.format(item['Director'])},
+            ':writer': {'S': '{}'.format(item['Writer'])},
+            ':actors': {'S': '{}'.format(item['Actors'])},
+            ':plot': {'S': '{}'.format(item['Plot'])},
+            ':lang': {'S': '{}'.format(item['Language'])},
+            ':country': {'S': '{}'.format(item['Country'])},
+            ':awards': {'S': '{}'.format(item['Awards'])},
+            ':poster': {'S': '{}'.format(item['Poster'])},
+    #        ':ratings': {'L': item['Ratings'], 'M': item['Ratings'][item]},
```

Comment:
Still need to figure out how to properly assign the type to this one.  Dict, list. str? Who knows!